modules/Hoot/managers/api.js

```diff
@@ -1111,8 +1111,9 @@ export default class API {
 
     grailPullOverpassToDb( data ) {
         const params = {
-            path: `/grail/pulloverpasstodb?bbox=${ data.BBOX }&name=${ data.secondaryName }`,
-            method: 'GET'
+            path: '/grail/pulloverpasstodb',
+            method: 'POST',
```

Comment:
changed some of these to post so we could send the customQuery through the data instead of in the url because smaller size limit there